packages/rum-react/src/domain/reactRouterV6/startReactRouterView.spec.ts

```diff
@@ -42,57 +43,64 @@ describe('computeViewName', () => {
     expect(computeViewName([] as RouteMatch[])).toBe('')
   })
 
-  it('returns the path of the first route match', () => {
-    expect(computeViewName([{ route: { path: '/foo' } }] as RouteMatch[])).toBe('/foo')
-  })
-
-  it('concatenates the paths of all route matches', () => {
-    expect(
-      computeViewName([
-        { route: { path: '/foo' } },
-        { route: { path: 'bar' } },
-        { route: { path: ':id' } },
-      ] as RouteMatch[])
-    ).toBe('/foo/bar/:id')
-  })
-
   it('ignores routes without a path', () => {
     expect(
       computeViewName([{ route: { path: '/foo' } }, { route: {} }, { route: { path: ':id' } }] as RouteMatch[])
     ).toBe('/foo/:id')
   })
 
-  it('handles absolute paths', () => {
-    expect(
-      computeViewName([
-        { route: { path: '/foo' } },
-        { route: { path: '/bar' } },
-        { route: { path: '/:id' } },
-      ] as RouteMatch[])
-    ).toBe('/:id')
-  })
+  // prettier-ignore
+  const cases = [
+      // route paths,         path,           expected view name
 
-  it('removes intermediary trailing slashes', () => {
-    expect(
-      computeViewName([
-        { route: { path: '/foo/' } },
-        { route: { path: 'bar/' } },
-        { route: { path: ':id/' } },
-      ] as RouteMatch[])
-    ).toBe('/foo/bar/:id/')
-  })
+      // Simple paths
+      ['/foo',                '/foo',         '/foo'],
+      ['/foo',                '/bar',         '/foo'], // apparently when the path doesn't match any route, React Router returns the last route as a matching route
+      ['/foo > bar',          '/foo/bar',     '/foo/bar'],
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏I don't know about the ">" syntax is it a react-router thing?

Reply:
It's not, it's just quick way to define nested routes for tests, see `.split(' > ')` below